docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,102 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Link between an user and a group.
+ * 
+ * @author jessedin
+ */
+@Entity
+@Table(name = "T_REVIEW")
+public class Review {
+
+    /**
+     * User ID.
+     */
+    @Id
+    @Column(name = "REV_IDUSER_C", length = 36)
+    private String userId;
+    
+    /**
+     * Document ID.
+     */
+    @Column(name = "REV_IDDOC_C", length = 36)
+    private String docId;
+
+    /**
+     * Rating.
+     */
+    @Column(name = "REV_RATING_C", length = 36)
+    private Integer rating;
+
+    /**
+     * Reviewer ID.
+     */
+    @Column(name = "REV_IDREVIEWER_C", nullable = false, length = 36)
```

Comment:
"REV_REVIEWERID_C" might be better since then variable in the next line is called "reviewerID"